prov/efa/test/efa_unit_tests.c

```diff
@@ -0,0 +1,14 @@
+#include "efa_unit_tests.h"
+int main(void)
+{
+	int ret;
+	const struct CMUnitTest getinfo_tests[] = {
+		cmocka_unit_test(efa_getinfo_default),
+	};
+	cmocka_set_message_output(CM_OUTPUT_XML);
+
+	ret = cmocka_run_group_tests_name("efa getinfo tests", getinfo_tests, NULL, NULL);
+	if (ret != 0)
```

Comment:
Is there any reason for this if statement?

Reply:
At some point I wrote an example that was supposed to be "tests that fail" and had this check, but I can remove it.